code/espurna/button.cpp

```diff
@@ -933,6 +993,12 @@ ButtonEvent Button::loop() {
         }
     }
 
+    if (ac_switch) {
+        if (ac_switch->loop()) {
+            return ac_switch->state() ? ButtonEvent::Pressed : ButtonEvent::Released;
+        }
+    }
+
```

Comment:
also iirc... see AnalogPin below? assume digitalRead() is the loop() call of the AcSwitch class (i.e. see DebounceEvent loop() logic and existing flow)